README.md

```diff
@@ -20,9 +23,123 @@ npm install
 poetry run python -m nltk.downloader punkt
 export ENV=local
 op inject --in-file .env.tpl --out-file .env
+poetry run python manage.py collectstatic --noinput
+poetry run python manage.py migrate
+poetry run python manage.py waffle_switch search-sort-radio-buttons off --create
 poetry run python manage.py runserver
 ```
 
+# Running the app against the RDS database(s)
+In order to run the app against the RDS database, you will need to do the following.
+1) Create a loop back pod for the given namespace.
+    ```
+    kubectl -n data-platform-find-moj-data-dev \
+    run port-forward-pod \
+    --image=ministryofjustice/port-forward \
+    --port=5432 \
+    --env="REMOTE_HOST=cloud-platform-2d5acdf1ab5379e3.cdwm328dlye6.eu-west-2.rds.amazonaws.com" \
+    --env="LOCAL_PORT=5432" \
+    --env="REMOTE_PORT=5432"
+    ```
+2) Forward traffic from your local host to the remote pod and keep the connection open. Note the local port.
+
+    ```
+    kubectl -n data-platform-find-moj-data-dev port-forward port-forward-pod 1234:5432
+    ```
+3) You can test connectivity as follows using postgres utility psql.
+
+    ```
+    psql postgres://< Database Username >:< Database Password >@localhost:1234/< Database Name >
+    ```
+    ```
+    psql (14.11 (Homebrew), server 16.3)
+    WARNING: psql major version 14, server major version 16.
+            Some psql features might not work.
+    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+    Type "help" for help.
+
+    db2d5acdf1ab5379e3=>
+    ```
+
+4) Via Netcat
+
+    ```
+    nc -z localhost 1234
+    ```
+    ```
+    Connection to localhost port 1234 [tcp/search-agent] succeeded!
+    ```
+
+5) Populate env file & add additional environment variables
+
+    ```
+    op inject --in-file .env.tpl --out-file .env
+    ```
+
+    ```
+    RDS_INSTANCE_ADDRESS=< 1pass >
+    DATABASE_NAME=< 1pass >
+    DATABASE_USERNAME=< 1pass >
+    DATABASE_PASSWORD=< 1pass >
+    ```
+
+
+6) Important note for running loop back connections on Mac OS.
+Ensure that your `RDS_INSTANCE_ADDRESS` variable is set to `docker.for.mac.host.internal`,
+otherwise it will fail to connect.
+
+  ```RDS_INSTANCE_ADDRESS=docker.for.mac.host.internal```
+
+7) Change the postgres port number in ```settings.py``` and addionally in ```./scripts/app-entrypoint.sh``` if you are running as a docker image, to match the value used for the loopback connection local port value.
```

Comment:
as in the `1234` port you've used for this command `kubectl -n data-platform-find-moj-data-dev port-forward port-forward-pod 1234:5432`

Reply:
I have added additional clarification around this.